src/main/java/study/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,51 @@
+package study.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_SEPARATOR = ",|:";
+    private static final String CUSTOM_SEPARATOR_REGEX = "//(.)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        if (text == null || text.isEmpty()) {
```

Comment:
```suggestion
        if (isBlank(text)) {
```
if 구문 내의 로직을 isBlank()로 분리하는 것은 의미 있을까?

Reply:
null체크 없이 isBlank()로만 하면 에러가 발생하지 않을까요?

Reply:
오해가 있었던 것 같아요.
`text == null || text.isEmpty()` 코드를 isBlank() 메서드로 분리해 보는 것은 어떨까라는 피드백이었어요.

Reply:
아 너무 낯익은 메서드명이여서 오해하였던 것 같습니다.